package.json

```diff
@@ -40,6 +40,7 @@
     "@rollup/plugin-json": "^4.1.0",
     "@vue/test-utils": "^1.1.2",
     "archiver": "^3.0.0",
+    "autoprefixer": "^10.2.4",
```

Comment:
We're not using this directly on the Web but it's needed when using the link for ODS